### DISCUSSION TITLE
Getting Failed to load module 'LinkKit' on CircleCI after updating it manually

### DISCUSSION THREAD

**Discussion by panncherry** [fastlane/fastlane#17537] [Q&A]
I replaced LinkKit framework with the latest release in my iOS project. CircleCI failed with exit code 65. Within CricelCI, it failed at FastLane scan with "Failed to load module LinkKit". What's causing the failure and any pointers how to fix it?
<img width="355" alt="Screen Shot 2020-11-01 at 5 59 49 PM" src="https://user-images.githubusercontent.com/17572436/97817711-0ae8b280-1c6c-11eb-87b2-1b04e56fb48f.png">

**Comment by rogerluan** [comment]
Does this compile in your machine?

Plaid has ben extremely unstable since v2.0.0, they've been publishing broken builds 😬 
Yesterday one of my team members said that after upgrading from Plaid v2 to v1, he couldn't get rid of this error:

![image](https://user-images.githubusercontent.com/8419048/98251619-af624180-1f57-11eb-9089-cd88cdf13263.png)

He tried deleting Pods, derived data, cleaning xcode, rebooting, everything you can think of. The only thing that finally solved the issue for him (apparently), was rebooting without ticking "reopen all windows upon restart" 🤯 

The issue is most likely on Plaid itself rather than on _fastlane_, I recommend checking other Plaid versions, upgrading it if possible, and/or try to stay on v1 for now 😞
